app/controllers/codelijsten/edit.js

```diff
@@ -361,137 +388,116 @@ export default class CodelijstenEditController extends Controller {
   }
 
   @action
-  async exportCodelist() {
-    const latestConceptScheme = await this.getConceptSchemeById(
-      this.conceptScheme.id
-    );
-    const codelistTtlCode =
-      await latestConceptScheme.modelWithConceptsAsTtlCode();
+  opposite(boolean) {
+    if (typeof boolean !== 'boolean') {
+      showErrorToasterMessage(
+        this.toaster,
+        this.intl.t('messages.error.valueMustBeBoolean', {
+          value: boolean,
+        })
+      );
+      return false;
+    }
 
-    downloadTextAsFile(
-      {
-        filename: this.getExportFileName(),
-        contentAsText: codelistTtlCode,
-      },
-      document,
-      window
-    );
+    return !boolean;
   }
 
   getExportFileName() {
-    return `codelijst-${this.conceptScheme.id}-${this.getIsoDate()}.ttl`;
-  }
-
-  getIsoDate() {
     const isoDate = new Date().toISOString();
+    const date = isoDate.slice(0, 10);
 
-    return isoDate.slice(0, 10);
+    return `codelijst-${this.dbConceptScheme.id}-${date}.ttl`;
   }
 
-  hasNoEmptyValuesInConceptList() {
-    return this.concepts.every((concept) => concept.label.trim() !== '');
+  async getConceptSchemeById(conceptSchemeId) {
+    try {
+      const conceptScheme = await this.store.findRecord(
+        'concept-scheme',
+        conceptSchemeId
+      );
+
+      return conceptScheme;
+    } catch (error) {
+      throw this.intl.t('constraints.couldNotGetConceptSchemeWithId', {
+        id: conceptSchemeId,
+      });
+    }
   }
 
-  isValidConceptSchemeName() {
-    return this.codelistName.trim() !== '' && !this.isDuplicateName;
+  conceptModelToListItem(model) {
```

Comment:
Don't think we need a check if the model is actually a `Concept`

Reply:
We could use `model.toString()` this will return `<mode:ModelName::IdOfTheModel>` add a `isModel()` method to each model and use a regex to filter out the modelname